crates/evm/fuzz/src/strategies/uint.rs

```diff
@@ -91,20 +95,40 @@ pub struct UintStrategy {
     fixtures_weight: usize,
     /// The weight for purely random values
     random_weight: usize,
+    /// Minimum bound for generated values
+    min_bound: U256,
+    /// Maximum bound for generated values
+    max_bound: U256,
+    /// Use logarithmic sampling for large ranges
+    use_log_sampling: bool,
```

Comment:
ideally this should be determined within strategy based on fuzzed type and not passed as a param

Reply:
changes this and tried preserving the original behaviour